core/setup.py

```diff
@@ -69,11 +69,11 @@
         "sqlparse>=0.2.3,<0.5",
         # ----
         # These are major-version-0 packages also maintained by dbt-labs. Accept patches.
-        "dbt-extractor~=0.5.0",
-        "minimal-snowplow-tracker~=0.0.2",
-        "dbt-semantic-interfaces~=0.5.0a2",
-        "dbt-common~=0.1.6",
-        "dbt-adapters~=0.1.0a2",
+        "dbt-extractor>=0.5.0,<=0.6",
+        "minimal-snowplow-tracker>=0.0.2,<0.1",
+        "dbt-semantic-interfaces<1.0",
```

Comment:
This will remove the ability to install pre-releases. That may be intentional. If it's not, it should be `dbt-semantic-interfaces<1.0.0a1`.

Reply:
```suggestion
        "dbt-semantic-interfaces<1.0.0a1",
```